content/installation/java.md

```diff
@@ -44,6 +44,7 @@ You can now run Cucumber [from the command line](/cucumber/api/#from-the-command
 If you are going to use the lambda expressions API (Java 8) to write the step
 definitions, add the following dependencies to `build.gradle`:
 
+## Gradle 4.10.3 or older
```

Comment:
Maybe also don't use headers here, but versions in **bold**, consistent with your other PR (#317)

Reply:
Sure thing. I only thought about using headers as it was easier to quickly navigate to the appropriate instructions based on the Gradle versions one uses from a table of contents, or similar